### PR TITLE
Add support for cdnUrl and code coverage reporting

### LINKED ISSUE #100: Add support for cdnUrl and report code coverage

### PR DESCRIPTION
**Pull request by ganemone**
Fixes #100

### CODE REVIEW

File: build/test-runtime.js [hunk 1]
```diff
@@ -17,8 +17,11 @@ module.exports.TestRuntime = function({
     const base = '.fusion/dist/test';
     const server = path.resolve(dir, base, 'server/server-main.js');
     const client = path.resolve(dir, base, 'client/client-main.js');
-    let command = require.resolve('unitest/bin/cli.js');
+    // let command = require.resolve('unitest/bin/cli.js');
```
Comment (KevinGrandon) [inline build/test-runtime.js]: Is it intentional to leave this commented?

File: test/index.js [hunk 2]
```diff
@@ -2,8 +2,9 @@
 require('./compiler/api');
 require('./compiler/errors');
 require('./cli/dev');
-require('./cli/test');
+// require('./cli/test');
```
Comment (KevinGrandon) [inline test/index.js]: Should this be uncommented?

File: build/test-runtime.js (hunk 1 above)
Comment (ganemone) [reply build/test-runtime.js]: good catch - updating

File: test/index.js (hunk 2 above)
Comment (ganemone) [reply test/index.js]: this should be just deleted

**Comment by KevinGrandon** [comment]
Feels like we might possibly be introducing false-positives here with the run-command, given that tests are failing in master. Looking into this more.

**Pull request merged**